invenio_cli/commands/assets.py

```diff
@@ -108,6 +140,16 @@ def link_js_module(self, path):
         assets_pkg = self._assets_pkg()
 
         steps = [
+            FunctionStep(  # Install dependent packages
+                func=self._npm_install_command,
+                args={"path": path},
+                message="Installing dependent packages..."
+            ),
+            FunctionStep(  # Run build script
+                func=self._build_script,
+                args={"module_pkg": module_pkg},
+                message="Building..."
+            ),
             FunctionStep(  # Create link to global folder
                 func=self._run_script,
```

Comment:
Might as well rename this in passing (`_run_script` is too generic):

`self._run_script` -> `self._link_dist_script`